src/nav/infrastructure.yml

```diff
@@ -734,6 +734,8 @@ pages:
     pages:
       - title: Introduction to Azure integrations 
         path: /docs/infrastructure/microsoft-azure-integrations/get-started/introduction-azure-monitoring-integrations
+      - title: Azure Arc New Relic 
```

Comment:
We don't typically label categories this way, even though I see it looks like a pattern in the other category. Are there alternative names we can use? For example, maybe 'Set up New Relic with Azure Arc'?

Reply:
@akristen, I made suggested change...